bin/scrape-atcoder

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/env python
+# pylint: disable=invalid-name
+
+"""This script scrapes Polban Atcoder leaderboard and outputs a JSON array
+containing the parsed data."""
+
+import json
+from re import findall
+import requests
+from bs4 import BeautifulSoup
+
+
+URL = "https://atcoder.jp/ranking/all?f.Affiliation=Bandung+State+Polytechnic"
+
+def get_username(cell):
+    """get_username takes a BeautifulSoup object containing a <td> cell from
+    Atcoder leaderboard and outputs the parsed username from the <a> tag."""
+    raw = cell.find("a", {"class": "username"}).get("href").strip()
+    assert raw[:7] == "/users/"
+
+    return raw[7:]
+
+def get_polban_rank(cell):
+    """get_polban_rank takes a BeautifulSoup object containing a <td> cell from
+    Atcoder leaderboard and returns the user's rank across Polban users."""
+    res =  findall(r'\d+', cell.get_text().strip())
+    assert len(res) == 2
+
+    return int(res[0])
+
+def get_global_rank(cell):
+    """get_global_rank takes a BeautifulSoup object containing a <td> cell from
+    Atcoder leaderboard and returns the user's rank across Atcoder
+    users."""
+    res = findall(r'\d+', cell.get_text().strip())
+    assert len(res) == 2
+
+    return int(res[1])
+
+result = []
+
+headers = {
+    "User-Agent": "cp-leaderboard (https://github.com/jtkencode/cp-leaderboard)"
+}
+
+req = requests.get(URL, headers)
+
+soup = BeautifulSoup(req.content, "html.parser")
+leaderboard_rows = soup.select(".table-responsive tbody tr")
+assert len(leaderboard_rows) > 0
+
+for row in leaderboard_rows:
+    columns = row.find_all("td")
+    # assert len(columns) == 7
```

Comment:
Yang ini boleh minta di-uncomment?

`assert`-`assert` ini tujuannya biar kalau tiba-tiba script scraping-nya fail (misal, struktur page-nya berubah, atau tiba-tiba scraper-nya diblok) script-nya bakal fail dan return error. 😉

Reply:
noted, already fix it at the new commit 👍🏻